.gitignore

```diff
@@ -23,3 +23,4 @@ coverage/
 
 # Storybook build files
 docs/docs
+.vscode/settings.json
```

Comment:
Please revert this as we want settings for vscode checked in :)

Reply:
Allright, guess I can't have my own status and activity bar colors for the project then :( It's helpful when context switching..